wled00/data/index.js

```diff
@@ -549,6 +549,7 @@ function populateInfo(i)
 	${inforow("Uptime",getRuntimeStr(i.uptime))}
 	${inforow("Free heap",heap," kB")}
   	${inforow("Estimated current",pwru)}
+	${ "battery" in i ? inforow("Battery level",i.battery.level," %") : ''}
```

Comment:
There should be no change in index.js because of usermod.
Adapt your addToJsonInfo() to format output for Info page (I know this will strip useful information from info object, but until Info page is reworked, this is the only proper way.)